meteor-app/imports/api/orders/orders.js

```diff
@@ -0,0 +1,42 @@
+import { Mongo } from 'meteor/mongo'
```

Comment:
You have created this file as `/imports/api/orders/orders.js`
I would prefer it to be named `/imports/api/orders/schema.js` which is more descriptive of its function, and the import statement for it will look like this:

`import Orders from '/imports/api/orders/schema'`

Which I think reads much better, rather than the duplication of 

`import Orders from '/imports/api/orders/orders'`